src/libs/actions/IOU.ts

```diff
@@ -6733,7 +6733,14 @@ function approveMoneyRequest(expenseReport: OnyxEntry<OnyxTypes.Report>, full?:
         total = expenseReport?.unheldTotal;
     }
     const optimisticApprovedReportAction = ReportUtils.buildOptimisticApprovedReportAction(total, expenseReport?.currency ?? '', expenseReport?.reportID ?? '-1');
-    const optimisticNextStep = NextStepUtils.buildNextStep(expenseReport, CONST.REPORT.STATUS_NUM.APPROVED);
+
+    const approvalChain = PolicyUtils.getApprovalChain(PolicyUtils.getPolicy(expenseReport?.policyID), expenseReport?.ownerAccountID ?? -1, expenseReport?.total ?? 0);
+    const isLastApprover = approvalChain[approvalChain.length - 1] === currentUserEmail ?? true;
```

Comment:
Don't we need a length check here? like we did earlier? `approvalChain.length > 0`

Reply:
it will default to `true` so we don't need to

Reply:
no because it defaults to `true`